README.rst

```diff
@@ -20,16 +21,18 @@ See the `Contributing Guidelines <./CONTRIBUTING.rst>`_.
 reStructuredText for PEPs
 =========================
 
-Original PEP source should be written in reStructuredText format,
+PEP source text should be written in reStructuredText format,
 which is a constrained version of plaintext, and is described in
-PEP 12.  Older PEPs were often written in a more mildly restricted
-plaintext format, as described in PEP 9.  The ``pep2html.py``
-processing and installation script knows how to produce the HTML
-for either PEP format.
+`PEP 12 <https://www.python.org/dev/peps/pep-0012/>`_.
+The ``pep2html.py`` processing and installation script knows
+how to produce the HTML for the PEP format.
 
-For processing reStructuredText format PEPs, you need the docutils
-package, which is available from `PyPI <https://pypi.org/>`_.
-If you have pip, ``pip install docutils`` should install it.
+To render the PEPs, you'll first need to install the requirements,
+(preferably in a fresh virtual environment):
+
+.. code-block:: console
+
+    python -m pip install -r requirements.txt
```

Comment:
```suggestion
.. code-block:: bash

    python -m pip install -r requirements.txt
```


`console` is for when you have a prompt beginning `$` with a command and then show the command's output (this same general rule applies using triple backticks in Markdown).

For example, this:

```rst
.. code-block:: console

        # console - ❌
        python -m pip install -r requirements.txt

.. code-block:: bash

        # bash - ✅
        python -m pip install -r requirements.txt


.. code-block:: console

        # console - ✅
        $ python -m pip install -r requirements.txt
        Requirement already satisfied: Pygments>=2.9.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from -r requirements.txt (line 2)) (2.11.2)
        Requirement already satisfied: Sphinx>=4.0.2 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from -r requirements.txt (line 3)) (4.4.0)
        ...

.. code-block:: bash

        # bash - ❌
        python -m pip install -r requirements.txt
        Requirement already satisfied: Pygments>=2.9.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from -r requirements.txt (line 2)) (2.11.2)
        Requirement already satisfied: Sphinx>=4.0.2 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from -r requirements.txt (line 3)) (4.4.0)
        ...
```

Renders (on GitHub) as:

<img width="1030" alt="image" src="https://user-images.githubusercontent.com/1324225/155191906-6380f37f-72d7-4a74-aa62-5e238d78c4c6.png">

Reply:
Ah, I didn't know `console` worked _only_ with console lines that contained the prompt. Updated all instances accordingly, thanks.